Error while checking build/O2Physics/o2 for 765af8357625f50ed872f8d75cc03988d89a1b50 at 2026-04-20 21:45:
```
## sw/BUILD/O2Physics-latest/log
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:539:79: error: 'McCollsExtra' is not a member of 'o2::aod'; did you mean 'MultsExtra'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:539:91: error: template argument 3 is invalid
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:539:31: error: '<expression error>' in namespace 'o2::soa' does not name a type
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:547:21: error: 'ColMCTrueTable' has not been declared
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:547:46: error: expected ',' or '...' before 'const'
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:659:21: error: 'ColMCTrueTable' has not been declared
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:659:46: error: expected ',' or '...' before 'const'
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:687:27: error: 'ColMCTrueTable' has not been declared
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:687:52: error: expected ',' or '...' before 'const'
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:750:27: error: 'ColMCTrueTable' has not been declared
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:750:52: error: expected ',' or '...' before 'const'
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:552:33: error: 'mcparticles' was not declared in this scope; did you mean 'particle'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:557:19: error: 'mcCollision' was not declared in this scope; did you mean 'lrmccollision'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:559:31: error: 'RecCols' was not declared in this scope; did you mean 'RecCol'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:566:28: error: 'RecTracks' was not declared in this scope
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:569:52: error: expected primary-expression before '>' token
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:569:54: error: expected primary-expression before ')' token
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:611:31: error: 'mfttracks' was not declared in this scope; did you mean 'lrmfttracks'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:625:35: error: 'besttracks' was not declared in this scope
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:630:62: error: expected primary-expression before '>' token
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:630:64: error: expected primary-expression before ')' token
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:640:35: error: 'mcparticles' was not declared in this scope; did you mean 'particle'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:547:37: error: unused parameter 'iterator' [-Werror=unused-parameter]
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:662:33: error: 'mcparticles' was not declared in this scope; did you mean 'particle'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:667:19: error: 'mcCollision' was not declared in this scope; did you mean 'lrmccollision'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:669:33: error: 'mcparticles' was not declared in this scope; did you mean 'particle'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:659:37: error: unused parameter 'iterator' [-Werror=unused-parameter]
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:690:18: error: 'mcCollision' was not declared in this scope; did you mean 'lrmccollision'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:694:33: error: 'mcparticles' was not declared in this scope; did you mean 'particle'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:700:33: error: 'RecCols' was not declared in this scope; did you mean 'RecCol'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:707:71: error: 'mcCollision' was not declared in this scope; did you mean 'lrmccollision'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:714:33: error: 'mcparticles' was not declared in this scope; did you mean 'particle'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:718:40: error: 'mcCollision' was not declared in this scope; did you mean 'lrmccollision'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:721:31: error: 'RecCols' was not declared in this scope; did you mean 'RecCol'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:728:69: error: 'mcCollision' was not declared in this scope; did you mean 'lrmccollision'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:731:28: error: 'RecTracks' was not declared in this scope
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:744:44: error: 'mcCollision' was not declared in this scope; did you mean 'lrmccollision'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:687:43: error: unused parameter 'iterator' [-Werror=unused-parameter]
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:753:18: error: 'mcCollision' was not declared in this scope; did you mean 'lrmccollision'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:757:33: error: 'mcparticles' was not declared in this scope; did you mean 'particle'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:763:33: error: 'RecCols' was not declared in this scope; did you mean 'RecCol'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:770:71: error: 'mcCollision' was not declared in this scope; did you mean 'lrmccollision'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:777:33: error: 'mcparticles' was not declared in this scope; did you mean 'particle'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:780:40: error: 'mcCollision' was not declared in this scope; did you mean 'lrmccollision'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:783:31: error: 'RecCols' was not declared in this scope; did you mean 'RecCol'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:790:69: error: 'mcCollision' was not declared in this scope; did you mean 'lrmccollision'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:793:28: error: 'mfttracks' was not declared in this scope; did you mean 'lrmfttracks'?
/sw/SOURCES/O2Physics/slc9_x86-64-slc9_x86-64/0/PWGCF/TwoParticleCorrelations/TableProducer/longrangeMaker.cxx:803:44: error: 'mcCollision' was not declared in this scope; did you mean 'lrmccollision'?
[0 more errors; see full log]
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/O2Physics/15874/765af8357625f50ed872f8d75cc03988d89a1b50/build_O2Physics_o2/pretty.html).

--- a/PWGLF/Tasks/GlobalEventProperties/flattenictyPikp.cxx
+++ b/PWGLF/Tasks/GlobalEventProperties/flattenictyPikp.cxx
@@ -27,6 +27,7 @@
 #include "Common/DataModel/Centrality.h"
 #include "Common/DataModel/EventSelection.h"
 #include "Common/DataModel/FT0Corrected.h"
+#include "Common/DataModel/McCollisionExtra.h"
 #include "Common/DataModel/Multiplicity.h"
 #include "Common/DataModel/PIDResponseTOF.h"
 #include "Common/DataModel/PIDResponseTPC.h"

--- a/PWGLF/Tasks/GlobalEventProperties/heavyionMultiplicity.cxx
+++ b/PWGLF/Tasks/GlobalEventProperties/heavyionMultiplicity.cxx
@@ -40,6 +40,7 @@
 
 #include <TH1.h>
 #include <THnSparse.h>
+#include <TMathBase.h>
 #include <TPDGCode.h>
 #include <TString.h>
 

--- a/PWGLF/Tasks/Nuspex/hadronnucleicorrelation.cxx
+++ b/PWGLF/Tasks/Nuspex/hadronnucleicorrelation.cxx
@@ -17,7 +17,6 @@
 #include "PWGCF/Femto3D/DataModel/singletrackselector.h"
 
 #include "Common/Core/RecoDecay.h"
-#include "Common/DataModel/Multiplicity.h"
 
 #include <CCDB/BasicCCDBManager.h>
 #include <CCDB/CcdbApi.h>
@@ -42,6 +41,7 @@
 #include <TH3.h>
 #include <TList.h>
 #include <TPDGCode.h>
+#include <TParticlePDG.h>
 #include <TString.h>
 
 #include <chrono>

--- a/PWGLF/Tasks/QC/v0cascadesqa.cxx
+++ b/PWGLF/Tasks/QC/v0cascadesqa.cxx
@@ -47,6 +47,7 @@
 #include <Framework/runDataProcessing.h>
 
 #include <TH1.h>
+#include <TH3.h>
 #include <TPDGCode.h>
 
 #include <array>

--- a/PWGLF/Tasks/Resonances/kstarqa.cxx
+++ b/PWGLF/Tasks/Resonances/kstarqa.cxx
@@ -51,7 +51,6 @@
 #include <THn.h>
 #include <TPDGCode.h>
 #include <TRandom.h>
-#include <TRandom3.h>
 
 #include <algorithm>
 #include <array>

